fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/sink/FlussSinkBuilder.java

```diff
@@ -0,0 +1,235 @@
+    public FlussSinkBuilder<InputT> setBootstrapServers(String bootstrapServers) {
+        this.bootstrapServers = bootstrapServers;
+        return this;
+    }
+
+    /** Set the database for the sink. */
+    public FlussSinkBuilder<InputT> setDatabase(String database) {
+        this.database = database;
+        return this;
+    }
+
+    /** Set the table name for the sink. */
+    public FlussSinkBuilder<InputT> setTable(String table) {
+        this.tableName = table;
+        return this;
+    }
+
+    /** Set the row type for the sink. */
+    public FlussSinkBuilder<InputT> setRowType(RowType rowType) {
+        this.tableRowType = rowType;
+        return this;
+    }
+
+    /** Set whether to ignore delete operations. */
+    public FlussSinkBuilder<InputT> setIgnoreDelete(boolean ignoreDelete) {
+        this.ignoreDelete = ignoreDelete;
+        return this;
+    }
+
+    /** Set target column indexes. */
+    public FlussSinkBuilder<InputT> setPartialUpdateColumns(int[] partialUpdateColumns) {
+        this.partialUpdateColumns = partialUpdateColumns;
+        return this;
+    }
+
+    /** Set target column indexes. */
+    public FlussSinkBuilder<InputT> setDataLakeFormat(DataLakeFormat lakeFormat) {
+        this.lakeFormat = lakeFormat;
+        return this;
+    }
+
+    /** Set shuffle by bucket id. */
+    public FlussSinkBuilder<InputT> setShuffleByBucketId(boolean shuffleByBucketId) {
+        if (!shuffleByBucketId) {
+            this.shuffleByBucketId = false;
+        }
+        return this;
+    }
+
+    /** Configure this sink to use upsert semantics. */
+    public FlussSinkBuilder<InputT> useUpsert() {
+        this.isUpsert = true;
+        return this;
+    }
+
+    /** Configure this sink to use append-only semantics. */
+    public FlussSinkBuilder<InputT> useAppend() {
+        this.isUpsert = false;
+        return this;
+    }
+
+    /** Set a configuration option. */
+    public FlussSinkBuilder<InputT> setOption(String key, String value) {
+        configOptions.put(key, value);
+        return this;
+    }
+
+    /** Set multiple configuration options. */
+    public FlussSinkBuilder<InputT> setOptions(Map<String, String> options) {
+        configOptions.putAll(options);
+        return this;
+    }
+
+    /** Set a FlussSerializationSchema. */
+    public FlussSinkBuilder<InputT> setSerializationSchema(
+            FlussSerializationSchema<InputT> serializationSchema) {
+        this.serializationSchema = serializationSchema;
+        return this;
+    }
+
+    /** Build the FlussSink. */
+    public FlinkSink<InputT> build() {
+        validateConfiguration();
+
+        Configuration flussConfig = Configuration.fromMap(configOptions);
+
+        FlinkSink.SinkWriterBuilder<? extends FlinkSinkWriter<InputT>, InputT> writerBuilder;
+
+        TablePath tablePath = new TablePath(database, tableName);
+        flussConfig.setString("bootstrap.servers", bootstrapServers);
```

Comment:
nit: Use `ConfigOptions.BOOTSTRAP_SERVERS.key()` instead of hard code the option key.